ext/redhat/facter.spec.erb

```diff
@@ -94,11 +94,11 @@ rm -rf %{buildroot}
 
 %files
 %defattr(-,root,root,-)
-%{_bindir}/cfacter
+%{_bindir}/facter
 %{_libdir}/libfacter.so*
 %{_includedir}/facter
 %if 0%{?rhel} >= 7 || 0%{?fedora}
-%{vendor_ruby}/cfacter.rb
+%{vendor_ruby}/facter.rb
```

Comment:
I'm debating whether we need a cfacter.rb to maintain the `--cfacter` option in puppet or if we treat the `--cfacter` option as only supported with "cfacter" (which Facter 3 is not).